packages/react-components/src/molecules/UsersList.tsx

```diff
@@ -38,32 +45,40 @@ const userStyles = css({
   alignItems: 'center',
 });
 const nameStyles = css({
+  fontSize: `${13.6 / perRem}em`,
   overflow: 'hidden',
   whiteSpace: 'nowrap',
   textOverflow: 'ellipsis',
 });
 
 interface UsersListProps {
   users: ReadonlyArray<
-    Pick<UserResponse, 'displayName'> &
-      Partial<Pick<UserResponse, 'id' | 'firstName' | 'lastName' | 'avatarUrl'>>
+    | Pick<
+        UserResponse,
+        'displayName' | 'firstName' | 'lastName' | 'avatarUrl' | 'id'
+      >
+    | ExternalAuthor
   >;
 }
 const UsersList: FC<UsersListProps> = ({ users }) => (
   <ul css={listStyles}>
     {users.map((user, i) => (
-      <li key={user.id ?? i} css={itemStyles}>
-        <Link
-          href={user.id && network({}).users({}).user({ userId: user.id }).$}
-        >
+      <li key={`author-${i}`} css={itemStyles}>
+        {!isExternalAuthor(user) ? (
```

Comment:
is it just me that finds negated harder to read than just putting the branches the other way around? :D just wondering if there's a reason since I think you've done this in other places as well